components/Snackbar/src/MDCSnackbarManager.h

```diff
@@ -147,6 +147,13 @@
  */
 @property(class, nonatomic, strong, nullable) UIFont *buttonFont;
 
+/**
+ The button class to set the snackbar button.
+ Note: The class must be a subclass of MDCSnackbarMessageViewButton.
+
+ Default is set to MDCSnackbarMessageViewButton.
+ */
```

Comment:
I wonder whether a delegate pattern would be cleaner here. I’m hesitant to add an api that encourages subclassing for what is likely just stylistic needs.

Reply:
moved to a delegate pattern.